orchagent/portsorch.cpp

```diff
@@ -3317,6 +3364,10 @@ void PortsOrch::doPortTask(Consumer &consumer)
 
                 }
 
+                if (!initHostTxReadyState(p)) {
+                        SWSS_LOG_ERROR("Failed to initialize host_tx_ready for  port %s ", alias.c_str());
```

Comment:
No need of logging ERR. Change it to void function

Reply:
taken care